release/staging-resources-cdk/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "staging-resources-cdk",
+  "version": "0.1.0",
+  "bin": {
+    "staging-resources-cdk": "bin/staging-resources-cdk.js"
+  },
+  "scripts": {
```

Comment:
I would like to see us add eslint to this package as well to help with formatting.